### PR TITLE
added maze movement and basic win rules

### PR DESCRIPTION
**Pull request by Chadyka**
Added maze movement logic, this should be fully inmplemented now.
Also implemented basic win condition, steps are counted throughout the game for highscore display.

### CODE REVIEW

**Comment by Chadyka** [comment]
[Maze movement](https://trello.com/c/806jWeNi/8-maze-movement)

File: public/js/maze.js [hunk 1]
```diff
@@ -2,8 +2,67 @@ window.addEventListener("DOMContentLoaded", (event) => {
...
+  } else if (e.keyCode == "40") {
+    posY < 10 ? posY++ : (posY = mazeHeight);
+    steps++;
+  } else if (e.keyCode == "37" && isGap(posY, posX - 1)) {
+    posX > 1 ? posX-- : (posX = 1);
+    steps++;
+  } else if (e.keyCode == "39" && isGap(posY, posX)) {
+    posX < 10 ? posX++ : (posX = mazeWidth);
+    steps++;
+  }
```
Comment (yankela1227) [inline public/js/maze.js:38]: what is this?  listener for arrows?

**Review by yankela1227 — APPROVED** [review]
Fantastic work!

**Comment by Chadyka** [comment]
[Win condition](https://trello.com/c/h3rVCJs8/10-win-condition)

**Pull request merged**